src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -120,15 +128,82 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
  * \details If \f$ \Phi_{kab} \f$ is the generalized
  * harmonic spatial derivative variable, then the derivatives of the
  * spatial metric are
- * \f{align}
- *      \partial_k g_{ij} &= \Phi_{kij}
- * \f}
+ * \f[
+ *      \partial_k g_{ij} = \Phi_{kij}
+ * \f]
  *
  * This quantity is needed for computing spatial Christoffel symbols.
  */
 template <size_t SpatialDim, typename Frame, typename DataType>
+void deriv_spatial_metric(
+    gsl::not_null<tnsr::ijj<DataType, SpatialDim, Frame>*> d_spatial_metric,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+// @}
+
+// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes time derivative of the spatial metric.
+ *
+ * \details Let the generalized harmonic conjugate momentum and spatial
+ * derivative variables be \f$\Pi_{ab} = -t^c \partial_c \psi_{ab} \f$ and
+ * \f$\Phi_{iab} = \partial_i \psi_{ab} \f$. As \f$ t_i \equiv 0 \f$, the time
+ * derivative of spatial metric is given by the spatial sector of the time
+ * derivative of the spacetime metric, i.e.
```

Comment:
I think you mean `... given by the time derivative of the spatial sector of the spacetime metric`.